src/olympia/reviewers/utils.py

```diff
@@ -172,7 +172,7 @@ def render_score(self, value):
 class PendingManualApprovalQueueTable(AddonQueueTable):
     addon_type = tables.Column(verbose_name=_('Type'), accessor='type', orderable=False)
     waiting_time = tables.Column(
```

Comment:
Rename this column and the associated `_get_waiting_time()`, `render_waiting_time()` methods (and update `default_order_by`), now that it's not really about waiting time ?

Reply:
https://github.com/mozilla/addons-server/pull/20151/commits/9fe51c57b89ed6af368a93ee03e67571aae2ea54